examples/acquiring_data/downloading_cutouts.py

```diff
@@ -92,23 +91,4 @@
 
 files = Fido.fetch(query)
 files.sort()
-
-#####################################################
-# Now that we've downloaded the files, we can create
-# a `~sunpy.map.MapSequence` from them.
-
-sequence = sunpy.map.Map(files, sequence=True)
-
-#####################################################
-# Finally, we can construct an animation in time from
-# our stack of cutouts and interactively flip through
-# each image in our sequence. We first adjust the plot
-# settings on each image to ensure the colorbar is the
-# same at each time step.
-
-for each_map in sequence:
-    each_map.plot_settings['norm'] = ImageNormalize(vmin=0, vmax=5e3, stretch=SqrtStretch())
-plt.figure()
-ani = sequence.plot()
```

Comment:
We lose one of our only gallery animations tho and I am sad to see that go.

Reply:
Other suggestions welcome - "Plotting an animation" seems like it's a single task that only needs one example though 😢  - we can fill up the future tutorials with them though!

Reply:
Yeah, it should go, we should check if we have an animation sequence example somewhere and not if, a new PR/issue to add one in future.

Reply:
https://docs.sunpy.org/en/stable/generated/gallery/plotting/arrayanimatorwcs.html#sphx-glr-generated-gallery-plotting-arrayanimatorwcs-py already exsists 👍